server/request.go

```diff
@@ -15,22 +15,20 @@ const (
 	authenticationSchemaName = "Signature"
 )
 
-var mysteriumApiUrl string
-
-func newGetRequest(path string, params url.Values) (*http.Request, error) {
+func newGetRequest(apiURI, path string, params url.Values) (*http.Request, error) {
```

Comment:
Since `mApi.discoveryAPIAddress` seems to be used  as `apiURL` param value in all calls, does it make sense to parametrize it?

Reply:
not sure about this place, since its functions and not member methods we should not depend on internal state. Should we hardcoded in apiURI, can it be considered a state or never changing constant..